### DISCUSSION TITLE
Why all of the APIs in `api.ts` are still used in the source but the file is put under `legacy` path?

### DISCUSSION THREAD

**Discussion by khoanguyen-yang** [Emurgo/yoroi-mobile#2087] [Q&A] [answered]
Hi, I am looking at the entire source code of Yoroi mobile wallet.

The APIs in `api.ts` such as submit transaction, get transaction history, etc. are still used in the main flows. So why it is put inside `legecy` folder? If they are actually deprecated, where can I find the latest implementation for those APIs?

Really looking forward to your reply.

**Comment by stackchain** [comment, marked as answer]
hi @khoanguyen-yang, putting it simply, migration from flow -> ts. It's legacy, not deprecated, we will be updating and moving all together with the re-structure that we are planning. You can find the latest updates of this API directly on its documentation: https://github.com/Emurgo/yoroi-graphql-migration-backend
